frontend/components/commons/header/global-actions/ValidateDiscardAction.vue

```diff
@@ -26,55 +26,79 @@
       v-model="allSelected"
       :disabled="!visibleRecords.length"
       class="list__item__checkbox"
-    ></base-checkbox>
-    <slot name="first" :selectedRecords="selectedRecords" />
+    />
+    <annotation-label-selector
+      v-if="datasetTask === 'TextClassification'"
+      :class="'validate-discard-actions__select'"
+      :multi-label="isMultiLabel"
```

Comment:
Pass correctly isMultiLabel prop, now it is always false because it is not connected to the real information only with the default status from the prop.